gradle.properties

```diff
@@ -3,4 +3,5 @@ kotlin.mpp.enableCInteropCommonization=true
 kotlin.mpp.stability.nowarn=true
 kotlin.native.ignoreDisabledTargets=true
 
-artifactVersion = 1.7.0
\ No newline at end of file
+artifactVersion = 1.8.0
```

Comment:
Non-snapshot version on `development`?

Reply:
i really should rename the default branch to `main` and create a proper development branch. Will do after squashing and merging this.